src/components/Employee.vue

```diff
@@ -0,0 +1,382 @@
+<template>
+  <div class="container">
+    <!-- top header: menu... -->
+    <!-- <i class="fab fa-medium">ádas</i> -->
+    <!-- <i class="far fa-envelope"></i> -->
+    <!-- table nhân viên -->
+    
+    <div class="row">
+      <table class="table table-striped">
+        <thead>
+          <tr>
+            <th>
+              <a href="#">Name</a>
+              <br />
+            </th>
+
+            <th>
+              <a href="#">Address</a>
+              <br />
+            </th>
+
+            <th>
+              <a href="#">Age</a>
+              <br />
+            </th>
+
+            <th>
+              <a href="#">Company</a>
+              <br />
+
+              <!-- <p>{{company}}</p> -->
+            </th>
+            <th>
+              <!-- <a>action</a> -->
+              <br />
+            </th>
+          </tr>
```

Comment:
chổ này e thay thế các thẻ a thành div hoặc span nhé, ở đây href không làm gì!
với lại thử tạo 1 biến header rồi for header này ra xem sao nha